editor/components/block-list/block.js

```diff
@@ -534,4 +533,5 @@ export default compose(
 			isLocked: !! templateLock,
 		};
 	} ),
+	withFilters( 'editor.BlockListBlock' ),
```

Comment:
Can you explain the reason for this reordering?

Reply:
I need to double check with Collaborative Editing. If it gets executed after connect, we can reuse props passed when filtering. Otherwise the get passed too late so we need to grab them using selectors twice.

Reply:
Yes, I can confirm it is correct. If `withFilters` would be first in the compose chain it would become the outermost component, which means React would render it first. As explained above, we want to have it last in the chain to make sure we have all other HOCs applied so we can take advantage of their modifications. This is how it makes the most sense for Collaborative Editing. Let's see how it works in general. We can refine later.